File: MODUL 4/Python/PRAK405-2310817220011-ReginaSilvaMaharatini.py
n1, n2 = map(int, input().split())
hasil, result = 0, 0

for i in range(n1):
    for j in range(i, -1, -1):
        print('({} * {})'.format(j + 1, n2), end='')
        if j > 0:
            print(' + ', end='')
    hasil += (i + 1) * n2
    print(' =', hasil)
    result += hasil
print(result)

File: MODUL 4/C/PRAK405-2310817220011-ReginaSilvaMaharatini.c
#include <stdio.h>
int main() {
    int n1, n2, hasil = 0, result = 0, i, j;
    scanf("%d %d", &n1, &n2);
    for (i = 0; i < n1; i++) {
        for (j = i; j >= 0; j--) {
            printf("(%d * %d)", j + 1, n2);
            if (j > 0) {
                printf(" + ");}}
        hasil += (i + 1) * n2;
        printf(" = %d\n", hasil);
        result += hasil;}
    printf("%d", result);
    return 0;}
